### DISCUSSION TITLE
Unable to Login to Admin

### DISCUSSION THREAD

**Discussion by be-studio** [orchidsoftware/platform#2141] [Q&A] [answered]
When I navigate to `/admin` on my local server installation of Orchid I get a Laravel error page with the message: 

```
LogicException
You must call one of in() or append() methods before iterating over a Finder.
(View: /.../vendor/orchid/platform/resources/views/fields/password.blade.php)
```

How do I resolve this issue?

**Comment by tabuna** [comment, marked as answer]
Hi @be-studio. This error is because the icons were not defined. 
Make sure there is at least one entry in the configuration file, for example:

```php
'icons' => [
    'orc' => \Orchid\IconPack\Path::getFolder(),
],
```

Don't forget to clear the cache after changing:

```php
php artisan config:clear
```

**Comment by be-studio** [comment]
Thank you @tabuna for the very fast and helpful response. That solved the issue.
